core/mondoo-email-security.mql.yaml

```diff
@@ -0,0 +1,251 @@
+    docs:
+      desc: The SPF record should not contain any unnecessary whitespace.
+      audit: Run the `dig -t TXT <domain>` command and verify that the SPF record does not contain any whitespace
+      remediation: |
+        Remove all excess whitespace from your SPF record.
+    refs:
+      - url: https://en.wikipedia.org/wiki/Sender_Policy_Framework
+        title: SPF Record
+  - uid: mondoo-email-security-spf-fail
+    title: SPF should be set to fail or soft fail all
+    mql: |
+      dns.params['TXT']['rData'].where(/v=spf1/).all(/all/)
+    docs:
+      desc: The SPF record should be set to soft fail all.
+      audit: Run the `dig -t TXT <domain>` command and verify that the SPF record is set to fail or soft fail all
+      remediation: |
+        The SPF record should end with all.
+    refs:
+      - url: https://www.m3aawg.org/sites/default/files/m3aawg-email-authentication-recommended-best-practices-09-2020.pdf
+        title: M3AAWG Email Authentication Recommended Best Practices (2020)
+  - uid: mondoo-email-security-spf-dns-record
+    title: Do not use deprecated SPF DNS Record Type
+    mql: dns.records.where(type == "SPF") == empty
+    docs:
+      desc: The SPF record should not use the deprecated SPF DNS Record Type.
+      audit: Run the `dig SPF <domain>` command and verify that the SPF record does not use the deprecated SPF DNS Record Type
+      remediation: |
+        Remove the deprecated SPF DNS Record Type from your SPF record.
+    refs:
+      - url: https://en.wikipedia.org/wiki/Sender_Policy_Framework#DNS_SPF_Records
+        title: DNS SPF Records
+  - uid: mondoo-email-security-dmarc
+    title: Ensure DMARC DNS entry exists
+    mql: dns("_dmarc."+asset.name).records != empty
+    docs:
+      desc: DMARC is a scalable mechanism by which a mail-originating organization can express domain-level policies and preferences for message validation, disposition, and reporting that a mail-receiving organization can use to improve mail handling. 
+      audit: Run the `dig _dmarc.<domain>` command and verify that the DMARC DNS entry exists
+      remediation: |
+        Add the _dmarc entry to you DNS zone file.
+    refs:
+      - url: https://www.m3aawg.org/sites/default/files/m3aawg-email-authentication-recommended-best-practices-09-2020.pdf
+        title: M3AAWG Email Authentication Recommended Best Practices (2020)
+  - uid: mondoo-email-security-dmarc-version
+    title: Ensure DMARC version 1
+    mql: dns("_dmarc."+asset.name).params['TXT']['rData'].all(/v=DMARC1/)
+    docs:
+      desc: Set the DMARC version in the TXT record to 1. 
+      audit: Run the `dig _dmarc.<domain>` command and verify that the DMARC TXT record contains `v=DMARC1`.
+      remediation: |
+        Add a TXT record to your DNS zone file with the following format:
+        
+        ```
+        <domain> IN TXT "v=DMARC1; p=reject; pct=100; rua=mailto:6407d6da13d24@ag.dmarcly.com; ruf=mailto:6407d6da13d24@fo.dmarcly.com; fo=1;"
```

Comment:
Unless this is a domain we really need here our examples should be lunalectric.com

Reply:
@tas50 thank you for the feedback.